ql/src/java/org/apache/hadoop/hive/ql/ddl/table/info/desc/formatter/TextDescTableFormatter.java

```diff
@@ -100,6 +102,32 @@ public void describeTable(HiveConf conf, DataOutputStream out, String columnPath
     }
   }
 
+  private void addPartitionTransformData(DataOutputStream out, Table table, boolean isOutputPadded) throws IOException {
+    String partitionTransformData = "";
```

Comment:
nit: can we name this partitionTransformOutput instead, or something similar?

Reply:
done